__tests__/context.test.ts

```diff
@@ -40,19 +35,28 @@ describe('should handle incompatabilities', async () => {
     // setup
     const clientTransport = getClientTransport('client');
     const serverTransport = getServerTransport();
+
+    interface ExtendedContext {
+      testctx: string;
+    }
+    const extendedContext: ExtendedContext = {
+      testctx: Math.random().toString(),
+    };
+
+    const ServiceSchema = createServiceSchema<ExtendedContext>();
```

Comment:
```suggestion
    const extendedContext = {
      testctx: Math.random().toString(),
    };

    const ServiceSchema = createServiceSchema(extendedContext);
```

or alternatively

```suggestion
    const extendedContext = {
      testctx: Math.random().toString(),
    };

    const ServiceSchema = createServiceSchema<typeof extendedContext>();
```

though i prefer the first

Reply:
I have changed it to `const ServiceSchema = createServiceSchema(extendedContext);`